services/director-v2/tests/conftest.py

```diff
@@ -123,6 +112,11 @@ def mock_env(monkeypatch: MonkeyPatch) -> None:
     monkeypatch.setenv("DIRECTOR_V2_DASK_SCHEDULER_ENABLED", "false")
     monkeypatch.setenv("DIRECTOR_V2_DYNAMIC_SCHEDULER_ENABLED", "false")
 
+    monkeypatch.setenv("REGISTRY_AUTH", "false")
```

Comment:
the issue with scopes and loops is analogous to monkeypatch-ing environs in non-function scopes. We live thatone for next PR